File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/Location.java
package com.porkandlager.travelonbudget.wires.models.beans;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tista on 11/12/16.
 */

public class Location {

    @SerializedName("lat")
    private float latitude;

    @SerializedName("lon")
    private float longitude;

    public float getLatitude() {
        return latitude;
    }

    public float getLongitude() {
        return longitude;
    }
}

File: app/src/main/java/com/porkandlager/travelonbudget/wires/models/beans/Location.kt
package com.porkandlager.travelonbudget.wires.models.beans

import com.google.gson.annotations.SerializedName

/**
 * Created by tista on 11/12/16.
 */

class Location {

    @SerializedName("lat")
    val latitude: Float = 0.toFloat()

    @SerializedName("lon")
    val longitude: Float = 0.toFloat()

}
